error line 25

##[error]fhem/fhem-controls-actions/master/action.yml (Line: 23, Col: 5): Unexpected value 'result'
##[error]fhem/fhem-controls-actions/master/action.yml (Line: 23, Col: 5): Unexpected value 'result'
##[error]Fail to load fhem/fhem-controls-actions/master/action.yml

--- a/action.yml
+++ b/action.yml
@@ -20,7 +20,7 @@ inputs:
     default: 'w'
 outputs:
   time: # id of output
-    result: 'If controls file was written'
+    description: 'If controls file was written'
 runs:
   using: 'node12'
   main: 'main.js'

